entry.go

```diff
@@ -75,30 +116,14 @@ func Diff(repo, ref1, ref2, dir string) ([]Entry, error) {
 		if err != nil {
 			return nil, err
```

Comment:
```suggestion
			return nil, fmt.Errorf("error retrieving git last change for %s: %w", fp, err)
```
I cannot make a suggestion for the error above... (I didn't actually try to compile this code)

Reply:
I took a stab at wrapping these remaining issues in fd0e620238de715963ac53f024dba5c22598443a.